aptos-move/framework/aptos-experimental/sources/test_function_values.move

```diff
@@ -1,4 +1,7 @@
 module aptos_experimental::test_function_values {
+    struct Funcs {
```

Comment:
For my understanding of the context: this is a test module to check that it is publishable, right? So we specifically do not want `#[test_only]`

Reply:
yes.